samples/struts/pet-store/pom.xml

```diff
@@ -27,16 +27,16 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <struts.version>6.0.3</struts.version>
-        <jackson.version>2.13.4</jackson.version>
+        <jackson.version>2.13.3</jackson.version>
```

Comment:
Was the downgrade intended? If so why?

Reply:
This was a mistake and fixed with latest version.